src/Modules/Guilds/Hooks/useProposal.ts

```diff
@@ -26,7 +27,10 @@ export const formatterMiddleware = (data: InitialProposal): Proposal => {
   // Add timeDetail
   const currentTime = moment();
   let differenceInMilliseconds = currentTime.diff(clone.endTime);
-  let timeDifference = moment.duration(differenceInMilliseconds).humanize();
+  let timeDifference =

```

Comment:
I think you should be able to implement the same logic by just using `moment.humanize()` by playing around with the [relative time thresholds](https://momentjs.com/docs/#/customization/relative-time-threshold/) and [rounding](https://momentjs.com/docs/#/customization/relative-time-rounding/) configurations. No need to refactor this if it takes a lot of work, but might be cleaner as we're using this same logic in multiple places. This also solves the issue of translations, as moment handles that.

Reply:
i think this would be difficult to coordinate with translations, right?
because we have to define the language moment uses